winui/Masked-TextBox/Formatting-Value.md

```diff
@@ -7,4 +7,135 @@ control: SfMaskedTextBox
 documentation: ug
 ---
 
-# Value mask formats in WinUI SfMaskedTextBox
\ No newline at end of file
+# Value mask formats in WinUI SfMaskedTextBox
```

Comment:
@Vignesh-SF3580 Formatting value in WinUI SfMaskedTextBox

Reply:
Changed the content.